src/components/lib/Questions.tsx

```diff
@@ -99,20 +150,32 @@ export abstract class Questions {
    * @param unit
    * @returns
    */
-  static getDateUnitDefaultValue(props: IQuestionData, unit: DATE_UNIT): string | undefined {
+  private static getDateUnitDefaultValue(
```

Comment:
Don't think this is used anywhere?

Reply:
Indeed it is not. See 6f0a1bf79bcd4a8f0bea82fc8ac031e4db6f3465.